src/calculateRentalCost.js

```diff
@@ -29,6 +29,21 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const bigDiscount = 50;
+  const smallDiscount = 20;
+  const pricePerDay = 40;
+
+  const totalAmount = days * pricePerDay;
+
+  if (days >= 7) {
+    return totalAmount - bigDiscount;
+  }
+
+  if (days >= 3) {
```

Comment:
The number of days could also change, let's move them to the variables

Reply:
done